apps/impulse/lib/screens/home_screen.dart

```diff
@@ -83,7 +68,7 @@ class _HomeScreenState extends State<HomeScreen> {
                 Navigator.push(
                   context,
                   MaterialPageRoute(builder: (context) => ProfileScreen(
-                    initialUser: user,
+                    initialUser: user.id.isNotEmpty ? user : null,
```

Comment:
If the userNotifier only returns a User when it's there, this should work, right? Otherwise, just resolve this message
```suggestion
                    initialUser: user,
```